compiler-core/src/language_server/tests/snapshots/gleam_core__language_server__tests__completion__local_variable_bit_string.snap

```diff
@@ -0,0 +1,34 @@
+---
+source: compiler-core/src/language_server/tests/completion.rs
+expression: "\nfn wibble() {\n  let assert <<h:1>> as i = <<1:1>>\n\n}\n"
+---
+fn wibble() {
+  let assert <<h:1>> as i = <<1:1>>
+|
+}
+
+
+----- Completion content -----
+h
```

Comment:
I think this is out of date or unused

Reply:
I'm afraid I do not understand, what exactly is out of date or unused?

Reply:
Removed!